docs/index.rst

```diff
@@ -29,6 +29,10 @@ spectra, typically the output of a multi-object spectrograph (e.g.,
 JWST NIRSpec), and includes viewers for 1D and 2D spectra as well as
 contextual information like on-sky views of the spectrograph slit.
 
+For users who are interested in using glue in contexts beyond those provided by
```

Comment:
I am not sure about this. How often is a user going to end up in this doc but wants glue not in the Jdaviz context?